src/app/legacy/containers/CpsRecommendations/RecommendationsPromo/index.jsx

```diff
@@ -118,7 +124,10 @@ const RecommendationsPromo = ({ promo, eventTrackingData }) => {
       }}
       enableGelGutters
     >
-      <StyledPromoWrapper data-e2e="story-promo-wrapper">
+      <StyledPromoWrapper
+        data-e2e="story-promo-wrapper"
+        isArticlePage={pageType === ARTICLE_PAGE}
```

Comment:
Instead of inline, might be better to have this as a const to fit in with our standards

```jsx
const isArticlePage = pageType === ARTICLE_PAGE
...
<StyledPromoWrapper
        data-e2e="story-promo-wrapper"
        isArticlePage
>
```